src/Components/Organisms/Footer/Footer.tsx

```diff
@@ -0,0 +1,108 @@
+import { Container, Grid, Link, Stack } from '@mui/material';
+
+const Footer = () => {
+  const mediaXM = {
```

Comment:
What does mean const name"media XM"? Maybe you could change it to be more descriptive?

Reply:
fixed